ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -166,6 +174,15 @@ FAN_SS_DWI:
   uses:
   - dishwasher_run_status
 
+FAN_STM:
+  description: "Basic fan with supply air temperature monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - STM
+  uses:
+  - supply_air_temperature_sensor
```

Comment:
You don't need to define this field explicitly if you are implementing STM.

Reply:
Removed Master_alarm and failed_alarm
- supply_air_temperature_sensor removed from uses of fan_stm